### PR TITLE
Hsaylor 9718 rc

### PR DESCRIPTION
**Pull request by Hsaylor**
Generated SDK release using :
API Defs tag `sdk-2021-09-03`
Generator tag `3.38.0`

Discovery V1 tag `sdk-2021-09-03`

# Service Changes
### Assistant V1
* `alt_text` property added to Image response type
​
### Assistant V2
* `session_start_time` and `state`  properties added to `MessageContextGlobalSystem` model
* `alt_text` property added to Image response type
* `location` property of `RuntimeEntity` model changed from required to optional
​
### Discovery V2
* Enum update for  `CreateProjectConstants`
​
### NLU
* Fix for `listClassificationsModels` through return type change and removal of unnecessary model
​
### STT
* More languages supported for next generation models
​
### TTS
* New voice models added
​
### CC, Disco V1, LT, NLC, PI, TA, VisRec V3 & V4
* No changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,10 +55,11 @@
     "@commitlint/config-conventional": "^8.2.0",
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/git": "^9.0.0",
-    "axios": "^0.21.1",
+    "axios": "^0.21.4",
     "browserify": "^16.5.2",
     "codecov": "^3.8.1",
     "concat-stream": "^2.0.0",
+    "cz-conventional-changelog": "^3.3.0",
```
Comment (apaparazzi0329) [inline package.json:62]: Why was this dependency added?

Comment (apaparazzi0329) [reply package.json:62]: Also, I believe there was an ask to upgrade the ibm-cloud-sdk-core dependency to the latest version 2.14.3. While the current notation would still install the latest version we don't want users the ability to install a version of the ibm-cloud-sdk-core that still uses a vulnerable version of axios so we should be specific

**Review by apaparazzi0329 — CHANGES_REQUESTED** [review]
Looks good. Requested a small change regarding our dependencies

File: package.json (hunk 1 above)
Comment (Hsaylor) [reply package.json:62]: `cz-conventional-changelog` was added as a fix to the commitizen pipeline. We recommend it's use but weren't including it. It made sense to add it while we were updating the `package.json`. I got confirmation from @repjarms that it should be included. 

Ah good catch, I updated our `axios` dependency past the vulnerable version, but missed the update on the cloud core itself

**Review by apaparazzi0329 — APPROVED** [review]
LGTM!

**Review by apaparazzi0329 — APPROVED** [review]
Changes LGTM

**Pull request merged**